.github/settings.yml

```diff
@@ -0,0 +1,17 @@
+_extends: github-apps-config-next
+
+branches:
+  - name: master
+    protection:
+      required_pull_request_reviews:
+        required_approving_review_count: 1
+        dismiss_stale_reviews: true
+        require_code_owner_reviews: false
+      required_status_checks:
+        strict: true
+        contexts:
+          - "ci/circleci: test"
```

Comment:
In https://github.com/Financial-Times/dotcom-page-kit/pull/701 there are additional contexts could also apply to this.

Reply:
You should only need test as it is dependent on `build` passing for it to be run. We can add the others for belts and braces though if you'd like.

Reply:
That makes sense